c2corg_ui/static/js/simplesearch.js

```diff
@@ -94,7 +94,7 @@ app.module.directive('appSimpleSearch', app.simpleSearchDirective);
  * @param {angularGettext.Catalog} gettextCatalog Gettext catalog.
  * @ngInject
  */
-app.SimpleSearchController = function($scope, $compile, $attrs, apiUrl, gettextCatalog, $templateCache) {
+app.SearchController = function($scope, $compile, $attrs, apiUrl, gettextCatalog, $templateCache, appAuthentication) {
```

Comment:
Please don't revert my changes :)
The "search" directive/controller are now named "simpleSearch".